game_gg.py

```diff
@@ -0,0 +1,98 @@
+
+
+import random
+n=random.randint(0,10)
+print("This is a guessing game where the user needs to guess a number from 0 to 10 inorder to win.")
+print("")
+print("The quicker the user guesses the number, the higher the number of points.")
+print("")
+print ("Following are the rules:")
+print("> Maximum number of guesses would be 8.")
+print("> After first 5 wrong tries, the user would be asked if they need a hint. ")
+print("> If user needs a hint, they shall be provided with a hint:")
+print(" - Even number or odd number")
+print(" - Number is less than 5 or more than 5.")
+print("> Game over if user is unable to guess at all.")
+print("")
+print("Points table:")
+print("1 guess used: 15 points")
+print("2 guesses used:13 points")
+print("3 guesses used:11 points")
+print("4 guesses used:9 points")
+print("5 guesses used:7 points")
+print("6 guesses used:5 points")
+print("7 guesses used:3 points")
+print("8 guesses used:1 point")
+print("")
+print("The score will be displayed in the end if the user has guessed the right number")
+
+a=int(input("Please guess the number between -1 and 11:"))
+
+
+b=1
+while b<9:
```

Comment:
Would it not be easier to use a for loop?

Reply:
I can try it again but won't that repeat commands which should only be displayed once